### PR TITLE
removed the backtick from relasenote

### PR DESCRIPTION
**Pull request by 7h3-3mp7y-m4n**
Version(s):
`4.12`

Issue:

[Issue #64739 ](https://github.com/openshift/openshift-docs/issues/64739)
Link to docs preview:

[preview](https://github.com/7h3-3mp7y-m4n/openshift-docs/blob/tester/release_notes/ocp-4-12-release-notes.adoc)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -2229,7 +2229,7 @@ sourceStrategy:
 
 * Previously, the Cluster DNS Operator used GO Kubernetes libraries that were based on Kubernetes 1.24 while {product-title} 4.12 is based on Kubernetes 1.25. With this update, GO Kubernetes API is v1.25.2, which aligns the Cluster DNS Operator with {product-title} 4.12 that uses Kubernetes 1.25 APIs. (link: https://issues.redhat.com/browse/OCPBUGS-1558[*OCPBUGS-1558*])
 
-* Previously, setting the `disableNetworkDiagnostics` configuration to `true` did not persist when the `network-operator` pod was re-created. With this update, the `disableNetworkDiagnostics` configuration property of network`operator.openshift.io/cluster` no longer resets to its default value after network operator restart. (link:https://issues.redhat.com/browse/OCPBUGS-392[*OCPBUGS-392*])
+* Previously, setting the `disableNetworkDiagnostics` configuration to `true` did not persist when the `network-operator` pod was re-created. With this update, the `disableNetworkDiagnostics` configuration property of network operator.openshift.io/cluster no longer resets to its default value after network operator restart. (link:https://issues.redhat.com/browse/OCPBUGS-392[*OCPBUGS-392*])
```
Comment (opayne1) [inline release_notes/ocp-4-12-release-notes.adoc:2232]: Hi! Thanks for the PR! Based on the [issue](https://github.com/openshift/openshift-docs/issues/64739) this should be:
```suggestion
* Previously, setting the `disableNetworkDiagnostics` configuration to `true` did not persist when the `network-operator` pod was re-created. With this update, the `disableNetworkDiagnostics` configuration property of network `operator.openshift.io/cluster` no longer resets to its default value after network operator restart. (link:https://issues.redhat.com/browse/OCPBUGS-392[*OCPBUGS-392*])
```
There was just a missing space between network and `operator.openshift.io/cluster`. The backticks should stay. See [this section](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#:~:text=Use-,backticks,-for%20all%20non) of the contributor guide for more information on backticks. 😄

**Review by opayne1 — COMMENTED** [review]
Hi! Thanks for your PR! I have just one comment for you! Let me know if you have any questions.

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (7h3-3mp7y-m4n) [reply release_notes/ocp-4-12-release-notes.adoc:2232]: okay thanks @opayne1  I'll do the changes ..

**Comment by 7h3-3mp7y-m4n** [comment]
I've implemented the changes , thanks @opayne1 :)

**Review by opayne1 — APPROVED** [review]
Lgtm! 

Preview here: https://67325--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes#ocp-4-12-bug-fixes

**Pull request merged**